src/components/StakingVaults/StakingTable.tsx

```diff
@@ -20,6 +24,7 @@ type StakingTableProps = {
 type RowProps = Row<EarnOpportunityType>
 
 export const StakingTable = ({ data, onClick, showTeaser }: StakingTableProps) => {
+  const assets = useAppSelector(selectAssets)
```

Comment:
we can avoid this, and passed in `row.original.assetId`, and have `makeDefiProviderDisplayName` do the suggested thing below